v1/pipeline/stages/iacm/execution-wrapper-config.yaml

```diff
@@ -11,6 +11,7 @@ properties:
     - $ref: ../../steps/common/action-step-node.yaml
     - $ref: ../../steps/iacm/iacmterraform-plugin-step-node.yaml
     - $ref: ../../steps/iacm/iacmapproval-step-node.yaml
+    - $ref: ../../steps/custom/template-step-node.yaml
```

Comment:
there is no change in corresponding v1 pipeline.json and v1 template.json
did we ran bundler again after making change ?

Reply:
Good catch, I did not actually run the bundler after that last change. Will do now and submit again the PR